main.tf

```diff
@@ -3,7 +3,7 @@
 //
 // ----------------------------------------------------------------------------
 terraform {
-  required_version = ">= 0.12.17"
+  required_version = ">= 0.13.0"
```

Comment:
Using features only in 0.13?

Reply:
No. Just given the CI is running against 0.13 now this feels the safest approach unless I run two CI stages for 0.12 and 0.13. 0.13 is in line with the latest jx builders. Thoughts?

Reply:
Feels a bit too early to stop supporting 0.12 imo. For eks, I think I will go with supporting 0.12 as long as I dont use a feature which is exclusive to 0.13. I see most modules supporting both versions, so ...
Two CI builds (matrix build of some sort) would be ideal, but there are resource constraints.

Reply:
It's fine to test with 0.13 only, but we should not force the users to upgrade to 0.13 yet. wdyt?

Reply:
Can go with 0.12 and CI both versions. Though the conditional validation on TLS variables in #22 might make this a moot point - I think that is using 0.13 only functionality. Will wait and see what that potential PR looks like first before I commit  either way

Reply:
Ok. See issue #22. I quite like the conditional validation here - so long as the variables are related and warrant residing within the same object I think this keeps cognitive load quite low. So we can either merge this one with 0.13 or merge the PR I'm about to open for #22 with 0.13. Your call :)

Reply:
I guess it's fine then. You can go with 0.13 (as it will be needed for #22) :+1: